src/index.js

```diff
@@ -19,7 +19,7 @@ const common = require('@google-cloud/common');
 const extend = require('extend');
 const GrpcService = require('@google-cloud/common-grpc').Service;
 const googleAuth = require('google-auto-auth');
-const grpc = require('google-gax').grpc().grpc;
+const gax = require('google-gax');
```

Comment:
Not sure if you use gax elsewhere in the file, but you could shorten this all to `const {GrpcClient} = require('google-gax');`

Reply:
Good call, changed